microsetta_admin/server.py

```diff
@@ -516,11 +518,31 @@ def new_kits():
         num_samples = int(request.form['num_samples'])
         prefix = request.form['prefix']
         selected_project_ids = request.form.getlist('project_ids')
+
```

Comment:
This comment applies to both Create Kit(s) and Add Barcode(s) to Kit(s) - if the user is uploading CSVs, we should make sure the number of rows being provided for any/all CSVs is correct before we even attempt to hit the API. E.g., if they uploaded a CSV with 20 Kit IDs and a CSV with 25 Barcodes, bounce back an error immediately.